MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/DirectionsActivity.java

```diff
@@ -68,22 +72,29 @@ protected void onCreate(Bundle savedInstanceState) {
     // Setup the MapView
     mapView = findViewById(R.id.mapView);
     mapView.onCreate(savedInstanceState);
-    mapView.getMapAsync(map -> {
-
-      DirectionsActivity.this.mapboxMap = map;
+    mapView.getMapAsync(new OnMapReadyCallback() {
+      @Override
+      public void onMapReady(@NonNull MapboxMap mapboxMap) {
+        DirectionsActivity.this.mapboxMap = mapboxMap;
 
-      // Set the origin location to the Alhambra landmark in Granada, Spain.
-      origin = Point.fromLngLat(-3.588098, 37.176164);
+        mapboxMap.setStyle(Style.MAPBOX_STREETS, new Style.OnStyleLoaded() {
+          @Override
+          public void onStyleLoaded(@NonNull Style style) {
+            // Set the origin location to the Alhambra landmark in Granada, Spain.
+            origin = Point.fromLngLat(-3.588098, 37.176164);
 
-      // Set the destination location to the Plaza del Triunfo in Granada, Spain.
-      destination = Point.fromLngLat(-3.601845, 37.184080);
+            // Set the destination location to the Plaza del Triunfo in Granada, Spain.
+            destination = Point.fromLngLat(-3.601845, 37.184080);
 
-      initSource();
+            initSource();
 
-      initLayers();
+            initLayers();
```

Comment:
add style as parameter to above 2 methods

Reply:
fixed